server/base/src/main/java/org/apache/accumulo/server/fs/VolumeChooserEnvironment.java

```diff
@@ -14,15 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.server.fs;
 
-import java.util.Objects;
+package org.apache.accumulo.server.fs;
 
-import org.apache.accumulo.core.clientImpl.Table;
-import org.apache.accumulo.server.ServerContext;
+import org.apache.accumulo.core.spi.common.ServiceEnvironment;
+import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.io.Text;
 
-public class VolumeChooserEnvironment {
+public interface VolumeChooserEnvironment {
```

Comment:
Do we still need a separate env for VolumeChooser now?  If so this could extend ServiceEnv

Reply:
Still need a separate env for VolumeChooser because it has method that ServiceEnv does not. 

It could extend.  I like having method instead because when a developer looks at the list of methods for volume chooser env then only see those related to volumes.